src/App.test.tsx

```diff
@@ -1,4 +1,3 @@
-import React from 'react'
 import { render, screen } from '@testing-library/react'
```

Comment:
Acho que esse arquivo pode ser deletado por agora. Ainda nao temos padrao de teste definido pro front.

Reply:
sim sim, quando for definido a gente volta ele